python/ray/experimental/state.py

```diff
@@ -545,6 +556,11 @@ def micros(ts):
         def micros_rel(ts):
             return micros(ts - start_time)
 
+        task_profiles = ray.global_state.task_profiles(start=0,
```

Comment:
this shouldn't call `ray.global_state.task_profiles`, because we are in `state.py`, so we should be `self.task_profiles`. This applies to